### ISSUE TITLE
Karma Chrome: logging not visible on Linux

### ISSUE THREAD

**Issue by vemv** [bensu/doo#158]
Hi!

Same exact configuration and install procedure on OSX and Linux gives me different results: on Mac console.log is visible, but not on Linux, which makes things harder to debug.

Hopefully it's an easy one, apparently there's a `'--enable-logging',` flag one can pass.

**Comment by bensu** [comment]
Thanks for the report! Can you try adding `--enable-logging`? If that works well, it'd be a small PR.

**Comment by vemv** [comment]
Hey there!

From what I see This would bit a fair bit of experimentation

![image](https://user-images.githubusercontent.com/1162994/32753853-ce4aedc8-c8ce-11e7-95ba-1cdb3582d2bc.png)

**Comment by vemv** [comment]
(I unintentionally hit Comment too early. Continuing here)

I can give it a shot, but I'd need to know how to have locally hackable copy of Doo attached to a real project.

How do I do that? Lein "checkout deps"?

Thanks - Victor

**Comment by miikka** [comment]
Yeah, [checkout dependencies](https://github.com/technomancy/leiningen/blob/master/doc/TUTORIAL.md#checkout-dependencies) is a good option. Or you can just run `lein install` in your local copy of Doo, you can then use `[lein-doo "0.1.9-SNAPSHOT"]` in the testing project. It's a bit more cumbersome, though: you need to run it in both `library` and `plugin` directories, and do it each time you change something.

**Comment by briangorman** [comment]
This seems resolved now
